disco_aws_automation/disco_elasticache.py

```diff
@@ -57,11 +57,16 @@ def update(self, cluster_name):
         Modifying tags, number of nodes, instance type, engine type, and port is not supported
         Args:
             cluster_name (str): name of cluster
+            maintenance_window(str): accept Preferred Maintenance Window value
+                                    or assigns defaul value from sun:1:00-sun:2:00.
         """
+        default_maintenance_window = "sat:5:00-sat:6:00"
```

Comment:
python style [guide](https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming). Also this should be global.